node-registrar/docs/swagger.yaml

```diff
@@ -1,5 +1,6 @@
+basePath: /v1
 definitions:
-  db.Account:
+  github_com_threefoldtech_tfgrid4-sdk-go_node-registrar_pkg_db.Account:
```

Comment:
Why did we move to that for resources?

Reply:
Yes, I noticed that, but it's auto-generated and seems unharmful.
I just rerun `swag init -g pkg/server/handlers.go --output docs --parseDependency --parseDepth 2` as I always did to update the swagger files since the node model was changed.
Not sure why that location was changed, but I believe that not caused by the changes in this PR? Maybe something related to moving the repo? Any Idea?